src/packages/core/src/server.ts

```diff
@@ -255,7 +255,9 @@ export class Server<
             `listen EADDRINUSE: address already in use ${
               host || DEFAULT_HOST
             }:${portNumber}.`
-          );
+          ) as any;
```

Comment:
If you like, this type assertion could be to `NodeJS.ErrnoException`, but I don't think I care strongly either way.

Reply:
fixed!